lib/features/family/features/reflect/data/gratitude_tags_data.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:givt_app/features/children/generosity_challenge/models/color_combo.dart';
+import 'package:givt_app/features/family/features/recommendation/tags/models/areas.dart';
+import 'package:givt_app/features/family/features/recommendation/tags/models/tag.dart';
+import 'package:givt_app/features/family/features/reflect/data/gratitude_category.dart';
+
+class GratitudeTagsData {
+  static final tags = [
+    const Tag(
+      key: 'AFTERADISASTER',
+      area: Areas.disaster,
+      displayText: 'After a disaster',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/aid.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'CAREFORCHILDREN',
+      area: Areas.health,
+      displayText: 'Child care',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/children.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'CLEANOCEANS',
+      area: Areas.environment,
+      displayText: 'Clean oceans',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/ocean.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'FINDAHOME',
+      area: Areas.basic,
+      displayText: 'Find a home',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/home.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'GETFOOD',
+      area: Areas.basic,
+      displayText: 'Get food',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/food.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'GOTOSCHOOL',
+      area: Areas.education,
+      displayText: 'Go to school',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/school.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'LEARNTOREAD',
+      area: Areas.education,
+      displayText: 'Learn to read',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/learn.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'PROTECTANIMALS',
+      area: Areas.environment,
+      displayText: 'Protect animals',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/panda.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'PROTECTFORESTS',
+      area: Areas.environment,
+      displayText: 'Protect forests',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/tree.svg',
+      type: TagType.INTERESTS,
+    ),
+    const Tag(
+      key: 'STAYHEALTHY',
+      area: Areas.health,
+      displayText: 'Stay healthy',
+      pictureUrl:
+          'https://givtstoragedebug.blob.core.windows.net/public/cdn/tag-logos/healthy.svg',
```

Comment:
Do we need the above svgs? I see you have urls here. Are the above svgs not FontAwesomeIcons?

Reply:
ill look up the icons